scripts/validate_phone_numbers.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+import json
+import sys
+
+# usage: AWS_PROFILE=notify-production python validate_phone_numbers.py numbers.txt
+# numbers.txt should have one phone number per line.
```

Comment:
Can we please add a comment that each run of the phone validation using pinpoint will incur a cost per phone number?